ScoreSaber/Core/Utils/Nine.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.ComponentModel;
+namespace System.Runtime.CompilerServices {
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class IsExternalInit {
+    }
+}
```

Comment:
Awful file name (unless this is actually required like this, but I doubt it)

Reply:
this was from umbra, im not sure what for